test/indexTest.js

```diff
@@ -152,3 +152,26 @@ test('key derivation', (t) => {
   t.deepEqual(nacl.sign.open(signed, key.publicKey), null, 'verification failure')
   t.throws(crypto.deriveSigningKeysFromSeed.bind(null, []), /Uint8Array/, 'error when input is not a Uint8Array')
 })
+
+test('signing', (t) => {
```

Comment:
please add the ed25519 test vector from https://datatracker.ietf.org/doc/rfc9421/ assuming this is the RFC we are using